### PR TITLE
added/edited few keymaps + clang-formatting

### PR DESCRIPTION
**Pull request by mayankdutta**
### changes: 
- while moving lines in visual mode, no  auto indentation was there, now is. 
- no highlighting changed to <leader> n
- <leader>h : for horizontal split 
- <leader>v : for vertical split.

### new: 
- <leader>t  : horizontal terminal
- <leader>T : vertical terminal

### CODE REVIEW

File: lua/lv-which-key/init.lua [hunk 1]
```diff
@@ -90,7 +108,11 @@ local mappings = {
     ["c"] = "Close Buffer",
     ["e"] = "Explorer",
     ["f"] = "Find File",
-    ["h"] = "No Highlight",
+    ["t"] = "split terminal",
```
Comment (RonaldsonBellande) [inline lua/lv-which-key/init.lua:115]: I like this addition

Comment (mayankdutta) [reply lua/lv-which-key/init.lua:115]: thanks !!

**Comment by mayankdutta** [comment]
Added `clang-formatting` option as mentioned in #563 .

**Review by jameswalmsley — APPROVED** [review]

**Review by jameswalmsley — APPROVED** [review]
I'm approving the clang format stuff.
I guess it should be separated from the key maps.. but I'm happy if it all gets merged..

**Comment by mayankdutta** [comment]
Should I generate seperate PR's?

**Pull request closed without merging**